### PR TITLE
Add getParseableZonedStringWithLocalTimeInOtherZone example to cookbook

### PR DESCRIPTION
**Pull request by ptomato**
See: #240

### CODE REVIEW

**Comment by ryzokuken** [comment]
@ptomato could you please rebase this on top of https://github.com/tc39/proposal-temporal/pull/506 or master if it gets merged by the time you read this? Thanks.

File: docs/cookbook/getParseableZonedStringWithLocalTimeInOtherZone.mjs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ *  return value
+ * @param {string} [sourceDisambguationPolicy=reject] - what to do when
+ *  `sourceDateTime` is ambiguous
+ * @returns {string} String indicating the time with time zone designation
+ */
+function getParseableZonedStringWithLocalTimeInOtherZone(
+  sourceDateTime,
+  sourceTimeZone,
+  targetTimeZone,
+  sourceDisambguationPolicy = 'reject'
```
Comment (Ms2ger) [inline docs/cookbook/getParseableZonedStringWithLocalTimeInOtherZone.mjs:23]:
```suggestion
  sourceDisambiguationPolicy = 'reject'
```

File: docs/cookbook/getParseableZonedStringWithLocalTimeInOtherZone.mjs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ * an error will be thrown by default.
+ * Usually this is what you want. (FIXME: but is it?)
+ * Use `sourceDisambiguationPolicy` to change this behaviour.
+ *
+ * @param {Temporal.DateTime} sourceDateTime - The local date and time
+ * @param {Temporal.TimeZone} sourceTimeZone - The time zone for
+ *  `sourceDateTime`
+ * @param {Temporal.TimeZone} targetTimeZone - The time zone for the
+ *  return value
+ * @param {string} [sourceDisambguationPolicy=reject] - what to do when
```
Comment (Ms2ger) [inline docs/cookbook/getParseableZonedStringWithLocalTimeInOtherZone.mjs:15]:
```suggestion
 * @param {string} [sourceDisambiguationPolicy=reject] - what to do when
```

File: docs/cookbook/getParseableZonedStringWithLocalTimeInOtherZone.mjs [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+ *  `sourceDateTime` is ambiguous
+ * @returns {string} String indicating the time with time zone designation
+ */
+function getParseableZonedStringWithLocalTimeInOtherZone(
+  sourceDateTime,
+  sourceTimeZone,
+  targetTimeZone,
+  sourceDisambguationPolicy = 'reject'
+) {
+  let instant = sourceDateTime.inTimeZone(sourceTimeZone, { disambiguation: sourceDisambguationPolicy });
```
Comment (Ms2ger) [inline docs/cookbook/getParseableZonedStringWithLocalTimeInOtherZone.mjs:25]:
```suggestion
  let instant = sourceDateTime.inTimeZone(sourceTimeZone, { disambiguation: sourceDisambiguationPolicy });
```

**Pull request merged**